{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "aca72ad6-4e40-4062-958e-f33940469a48",
   "metadata": {},
   "source": [
    "# MEGA\n",
    "\n",
    "- Input Data: FASTA file containing the sequences of same lenght.\n",
    "- Sequence Alignment: Selecting the alignment method (ClustalW) within MEGA to align the sequences.\n",
    "- Inspect Alignment: Review the alignment to ensure it's accurate and edit it if needed.\n",
    "- Construct Phylogenetic Tree: Maximum Parsimony to build the phylogenetic tree based on the aligned sequences.\n",
    "- Tree Visualization: Once the tree is constructed, visualize and explore the tree within MEGA. Export as .png image\n",
    "\n",
    "\n",
    "Inspecting the alignment in MEGA allows you to review the aligned sequences to ensure accuracy and make any necessary edits. Here's how you can inspect the alignment within MEGA: once you've aligned the sequences using methods like ClustalW or Muscle within MEGA, the aligned sequences will be displayed in the alignment view. MEGA provides a graphical interface to visualize the alignment. You'll see the aligned sequences displayed horizontally or vertically, with gaps (\"-\") indicating insertions or deletions in the alignment. Save or Export.\n",
    "\n",
    "We'have saved the alignment in FASTA format and the phylogenetic tree in .mdsx, .png and Newick format.\n",
    "\n",
    "### Analysis\n",
    "\n",
    "The provided phylogenetic tree illustrates evolutionary relationships among sequences, primarily based on their 18S ribosomal RNA gene sequences. It exhibits a nested hierarchical structure showcasing relationships among distinct species or sequences. \n",
    "Each sequence is denoted by an accession number accompanied by a species name, and comma-separated entries within parentheses signify clustering based on sequence similarity. Notably, the tree delineates distinct clades that imply relatedness and shared ancestry among sequences. The branching points in the tree signify evolutionary divergence and delineate the points of divergence among the represented species, it presents a rooted tree structure displaying multiple branching patterns that indicate evolutionary paths and divergences.\n",
    "\n",
    "This representation hints at groupings of sequences sharing common genetic traits, indicating evolutionary closeness, the visualization portrays an organized structure that highlights the genetic proximity among various species or sequences."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1f0e421e-94a8-4481-b50a-63dfac872cd7",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
